Sprint 1 PD/.env

```diff
@@ -0,0 +1,8 @@
+DATABASE_URL=jdbc:mysql://donnadb.cvaai4e089sw.us-east-2.rds.amazonaws.com:3306
```

Comment:
do not directly upload ".env" file, instead uploading ".env_example" file and put something like:
DATABASE_URL=
...
in ".env_example" file